src/__test__/operate.test.js

```diff
@@ -0,0 +1,19 @@
+import operate from '../logic/operate';
```

Comment:
On the operate tests you are missing to test the `%` operation and the `÷` by `0`. Please add the necessary code to cover all the functionalities & scenarios.